src/main/java/blitz/rijks/ApiKey.java

```diff
@@ -0,0 +1,39 @@
+package blitz.rijks;
+
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * Retrieve the API Key in apikey.properties or from environment variable on GitHub
+ */
+public class ApiKey {
```

Comment:
You don't need this anymore since I uploaded the code and you added it as a dependency.